showpass/Showpass-E2E-tests
# Add a spec to register a new user

**Description:**
Create a test script to verify that a new Showpass account can get created.

- Navigate to [Create Account](https://beta.showpass.com/accounts/signup/) 
- Populate the form using valid input
- Click ‘Create Account’
- Make sure a new account has been created

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-3

**How to test:**
Run the following script and make sure it passes "cypress/e2e/registration/register-new-user.cy.js"

**Test execution results:**
The script is currently disabled (marked as "skip") but can always be enabled back.

<img width="1715" alt="createAccount-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/69248244-dd74-4e02-b9da-67ad29cccf63">
